crescent/ext/tasks/loop.py

```diff
@@ -28,6 +28,57 @@ def _call_next(self) -> None:
         super()._call_next()
         self.first_loop = False
 
+    @overload
+    def set_interval(self, *, hours: int = ..., minutes: int = ..., seconds: int = ...) -> None:
+        ...
+
+    @overload
+    def set_interval(self, timedelta: _timedelta, /) -> None:
+        ...
+
+    def set_interval(
+        self,
+        timedelta: _timedelta | None = None,
+        *,
+        hours: int = 0,
+        minutes: int = 0,
+        seconds: int = 0,
+    ) -> None:
+        """
+        Cancel the currently scheduled task and schedule the next task and future tasks
+        with a new wait time.
```

Comment:
Sounds good. Perhaps a more concise wording:
> Stop the task, and restart it with a new interval.

But how it is right now is good. I say merge it with whatever wording you want.

If you're referring to the wording of `set_interval`, I can't think of a better alternative, so I say go with it.

Reply:
I prefer my wording, I feel my extra words make it more clear.